BatteryLimitMessage.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BatteryManagement
+{
+  class BatteryLimitMessage 
+  {
+        public static List<String> MeasureCrossedMaximum = new List<String>();
+        public static List<String> MeasureCrossedMinimum = new List<String>();
+        public static List<String> MeasureReachingLow = new List<String>();
+        public static List<String> MeasureReachingHigh = new List<String>();
+    
+        public static void CreateConsolidatedReportDE()
```

Comment:
Method name is Create***** but just invokes Print************ methods

Reply:
Renamed the method name as CallPrintMethods